libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -300,7 +300,10 @@ func (m *Manager) Apply(pid int) error {
 		return err
 	}
 
-	<-statusChan
+	select {
+	case <-statusChan:
+	case <-time.After(time.Second):
```

Comment:
Should have some sort of warning at least. I would also recommend investigating *why* we don't get the status from systemd. In the issue you said:

> Not sure how dbus works from within the containerized env or how we might fix this.

I'm not sure what containerised environment you're using, but dbus should be running as a daemon inside whatever container you're spawning `runc` inside as it normally would on your host. Can you reproduce the issue using `runc` on the host?

Reply:
@cyphar added warning. 
> what containerised environment ...

It occured in containerized OpenShift. I am trying to reproduce it on my local machine. Will update on that further.

Reply:
I believe the way this has worked in the past is that the origin node container runs as privileged, which implies a shared IPC and PID namespace with the host such that the containerized node can get dbus signals from the host dbus.